base_address_parish/views/res_partner_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="parish_view_partner_form" model="ir.ui.view">
+      <field name="name">parish.partner.view</field>
+      <field name="model">res.partner</field>
+      <field name="inherit_id" ref="base.view_partner_form" />
+      <field name="arch" type="xml">
+        <field name="street2" position="after">
+          <field
+                    name="parish_id"
+                    placeholder="Parish..."
+                    class="o_address_street"
+                    options="{'no_open': True, 'no_quick_create': True}"
```

Comment:
Parish Records are static only if changed territorial organization of country is need create a new record. So, i think users can not create new records here, it must be do in apropied menu
```suggestion
                    options="{'no_open': True, 'no_create': True}"
```